Signal/translations/en.lproj/Localizable.strings

```diff
@@ -547,6 +547,9 @@
 /* Error message when attempting to send message */
 "ERROR_DESCRIPTION_UNREGISTERED_RECIPIENT" = "Contact is not a Signal user.";
 
+/* Error message when unable to receive an attachment because the sending client is too old. */
+"ERROR_MESSAGE_ATTACHMENT_FROM_OLD_CLIENT" = "Failure: Ask sender to update Signal and resend.";
```

Comment:
Will this always be two lines when translated?

Reply:
I don't know, but the layout looks ok with only one line, and it resizes text so longer text will work too - though it gets pretty small. 

This should be a rarely seen piece of UI, so making it super flexible doesn't warrant much priority IMO.